drivers/interrupt_controller/intc_esp32.c

```diff
@@ -260,7 +260,11 @@ int esp_intr_reserve(int intno, int cpu)
 /* Returns true if handler for interrupt is not the default unhandled interrupt handler */
 static bool intr_has_handler(int intr, int cpu)
 {
-	return (intr_alloc_table[intr * CONFIG_MP_NUM_CPUS + cpu].handler != default_intr_handler);
+	bool r;
+
+	r = intr_alloc_table[intr * CONFIG_MP_MAX_NUM_CPUS + cpu].handler != default_intr_handler;
```

Comment:
Was there any compiling warning in here as well?

Reply:
It was line length.  So split it out this way.